modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughPopover.testcase

```diff
@@ -75,4 +95,28 @@ definition {
 		}
 	}
 
+	@description = "LPS-154640. Assert popover mode can have a shadow around element."
+	@priority = "2"
+	test CanHaveAShadowAroundElement {
+		task ("When click hotspot element until Step 4") {
+			Click(locator1 = "Walkthrough#HOTSPOT");
+
+			for (var number : list "2,3,4") {
+				Click(locator1 = "Walkthrough#NEXT_POPOVER");
+
+				AssertElementPresent(
+					key_title = "Step ${number} of 5: Title ${number}",
+					locator1 = "Walkthrough#POPOVER_TITLE");
```

Comment:
I think this could be useful converted into a macro to click through multiple steps

Reply:
Actually, we don't need to have the assertion at all here. We assert this in the other test `CanDisplayStepNumber` 

This is probably where I'd recommend removing the for loop and just have `Click(locator1 = "Walkthrough#NEXT_POPOVER");` three times.